setup.py

```diff
@@ -1313,12 +1333,13 @@ def main() -> None:
                 "lib/*.lib",
             ]
         )
-        aotriton_image_path = os.path.join(lib_path, "aotriton.images")
-        aks2_files = []
-        for root, dirs, files in os.walk(aotriton_image_path):
-            subpath = os.path.relpath(root, start=aotriton_image_path)
-            for fn in files:
-                aks2_files.append(os.path.join("lib/aotriton.images", subpath, fn))
+        # XXX: Why not use wildcards ["lib/aotriton.images/*", "lib/aotriton.images/**/*"] here?
+        aotriton_image_path = TORCH_DIR / "lib" / "aotriton.images"
+        aks2_files = [
+            file.relative_to(TORCH_DIR).as_posix()
+            for file in aotriton_image_path.rglob("*")
+            if file.is_file()
+        ]
```

Comment:
The previous code gets `aks2_files = []` and breaks the ROCm build on trunk. I'm not sure why.

I think the following code seems equivalent.

```diff
  aotriton_image_path = TORCH_DIR / "lib" / "aotriton.images"
- aks2_files: list[str] = []
- for file in filter(lambda p: p.is_file(), aotriton_image_path.glob("**")):
-     subpath = file.relative_to(aotriton_image_path)
-     aks2_files.append(os.path.join("lib/aotriton.images", subpath))
+ aks2_files = [
+     file.relative_to(TORCH_DIR).as_posix()
+     for file in aotriton_image_path.rglob("*")
+     if file.is_file()
+ ]
```

PS: `path.rglob("*")` is equivalent to `path.glob("**/*")`. `path.glob("**")` will yield `path` while `path.glob("**/*")` will not. But we are looking for files only, so they are the same.